jihyeon/app.js

```diff
@@ -32,10 +32,32 @@ app.get('/ping', function (req, res, next){
     return res.status(200).json({message: 'pong'})
 });
 
+app.post('/users/signup', async function (req, res) {
+    const { email, password, name, age, phone_number } = req.body
```

Comment:
```suggestion
    const { email, password, name, age, phoneNumber } = req.body
```
자바스크립트에서는 snake_case(밑줄로서 단어를 잇는)가 아닌 camelCase(소문자로 시작하여 대문자를 섞는 방법)를 사용하는것이 컨벤션입니다.

Reply:
camelCase로 쓰면 Postman에서 오류가 뜨는데 다른것도 같이 수정해야 하나요?